app/models/signup.rb

```diff
@@ -17,15 +18,20 @@ class Signup < ApplicationRecord
   validates :phone_number, phone: true, allow_blank: true, format: { with: /\A\+1[0-9]{10}\z/ }
   validates :email_address, 'valid_email_2/email': true
 
-  def self.valid_emails
-    distinct(:email_address).pluck(:email_address).filter do |email|
-      ValidEmail2::Address.new(email).valid?
+  def self.valid_emails_with_unsent_followups
+    distinct(:email_address).where(sent_followup: false).pluck(:email_address, :name).filter do |signup_info|
```

Comment:
I find it a bit unexpected that the return value of this query is a multi-dimensional array instead of a signup object. I wonder if we could remove the pluck and have it return signup objects. Then we can call for signup_info.email and signup_info.name in both this method and the send_followup_emails method!